src/actors/symbolication.rs

```diff
@@ -1405,19 +1423,34 @@ impl SymbolicationActor {
 
                                 let module_id = code_module.id();
 
-                                let module_cfi_status = match module_id {
-                                    Some(id) => {
-                                        unwind_statuses.get(&id).copied().unwrap_or_default()
-                                    }
+                                let unwind_status = match module_id {
+                                    Some(id) => match unwind_statuses.get(&id) {
+                                        // The code module was not in the original set of referenced
+                                        // modules, but was now required for stack walking.
+                                        None if scanned_modules.contains(&id) => {
+                                            ObjectFileStatus::Missing
+                                        }
```

Comment:
I think this would be a bug, could we get a sentry error event for this?  Probably just `log::error!(...)`?